draft-ietf-httpbis-resumable-upload.md

```diff
@@ -336,6 +336,8 @@ The client MUST NOT perform multiple upload transfers for the same upload resour
 
 If the offset indicated by the `Upload-Offset` field value does not match the offset provided by the immediate previous offset retrieval ({{offset-retrieving}}), or the end offset of the immediate previous incomplete successful transfer, the server MUST respond with a `409 (Conflict)` status code.
 
+The server applies the patch document of the `application/partial-upload` media type by appending the unmodified request content to the targeted upload resource.
```

Comment:
To me "unmodified" reads as "take the entire thing and apply it without changes" which is probably not what is intended here? The server is not allowed to pick bytes that it appends, but I think we need to clarify that the entire request must not complete for the server to append data. I.e. if the connection breaks down after 100 bytes then as much as possible of those 100 bytes _should_ be appended and what is appended _must_ be appended without modification.

Reply:
Good idea, thank you.